aptos-move/framework/aptos-stdlib/sources/hash.move

```diff
@@ -191,4 +203,32 @@ module aptos_std::aptos_hash {
             i = i + 1;
         };
     }
+
+
+    #[test(fx = @aptos_std)]
+    fun blake2b_256_test(fx: signer) {
```

Comment:
n00b question: do we have a test that ensures a failure when the feature flag is not enabled?

Reply:
no, could be added

Reply:
I added one.